src/main/target/NUCLEOF446RE/target.h

```diff
@@ -99,6 +99,7 @@
 #define SPI1_MOSI_PIN           PA7 // D11
 
 #define USE_RX_NRF24
+#define USE_RX_XN297  // Required for USE_RX_CX10
```

Comment:
The way this is done for other SPI RX is that the hardware drivers are enabled in `common_post.h` if the respective RX protocol drivers are defined. This reduces the complexity of the targets.